moto/core/models.py

```diff
@@ -702,7 +703,7 @@ def urls(self):
         A dictionary of the urls to be mocked with this service and the handlers
         that should be called in their place
         """
-        url_bases = self._url_module.url_bases
+        url_bases = self.url_bases
```

Comment:
Curious why this change was needed...

Reply:
It wasn't needed, just a simplification, as `self.url_bases` points to `self._url_module.url_bases`